lib/ansible/modules/source_control/git_config.py

```diff
@@ -64,6 +64,13 @@
         set that setting to the given value.
     required: false
     default: null
+  state:
```

Comment:
All of the other parameters in this list are in alphabetical order. Please ensure the new `state` parameter is too.

Reply:
Understood, will rearrange in the next commit